corehq/apps/data_interfaces/forms.py

```diff
@@ -270,6 +271,11 @@ class CaseRuleCriteriaForm(forms.Form):
         label=gettext_lazy("Case Type"),
         required=True,
     )
+    criteria_operator = forms.ChoiceField(
+        label=gettext_lazy("Critera Operator"),
```

Comment:
This text is rather obscure. One option is to leave it off altogether, since the select options are complete sentences. Another is to label is something like "Run when" and then change the options to "ALL of the criteria are met" / "ANY of the criteria are met".

Reply:
+1 to have `Run when`

Reply:
Yeah agreed this is much better